css/welcome.css

```diff
@@ -323,6 +333,11 @@ main {
     font-size: 1.8vmin;
   }
 
+  /* hide the text and let the card imgs read out the alt text */
+  #welcome-nav a span {
+    display: none;
+  }
+
```

Comment:
this may not be needed - can just delete the text span

Reply:
I think we still need this for the text to show in the mobile view. I can revise the above comment to clarify